wdn/templates_5.0/includes/global/search.html

```diff
@@ -1,18 +1,20 @@
 <div class="dcf-search dcf-header-global-item dcf-flex-grow-1 dcf-d-flex dcf-jc-flex-end dcf-d-none@print" id="dcf-search" role="search">
   <div class="dcf-search-toggle-wrapper dcf-ai-center dcf-w-100%">
-    <a href="https://search.unl.edu/" class="dcf-nav-toggle-btn dcf-nav-toggle-btn-search dcf-search-toggle dcf-d-flex dcf-ai-center dcf-jc-between dcf-w-100% dcf-p-0 dcf-txt-2xs dcf-bg-transparent dcf-b-1 dcf-b-solid unl-b-scarlet unl-font-sans unl-scarlet" id="dcf-search-toggle-link">
+    <a id="dcf-search-toggle-link" href="https://search.unl.edu/" class="dcf-d-flex dcf-ai-center dcf-jc-between dcf-w-100% dcf-p-0 dcf-txt-2xs dcf-bg-transparent dcf-b-1 dcf-b-solid unl-b-scarlet unl-font-sans unl-scarlet">
       <span class="dcf-search-toggle-label dcf-flex-grow-1 dcf-txt-left">Search</span>
       <span class="dcf-d-flex dcf-ai-center dcf-jc-center dcf-w-8 dcf-pt-3 dcf-pb-3 unl-bg-scarlet unl-cream">
         <svg class="dcf-h-4 dcf-w-4 dcf-fill-current" aria-hidden="true" focusable="false" height="16" width="16" viewBox="0 0 48 48">
           <path d="M18 36a17.9 17.9 0 0 0 11.27-4l15.31 15.41a2 2 0 0 0 2.84-2.82L32.08 29.18A18 18 0 1 0 18 36zm0-32A14 14 0 1 1 4 18 14 14 0 0 1 18 4z"></path>
         </svg>
       </span>
     </a>
+    <button type="button" disabled hidden aria-hidden="true" class="dcf-search-toggle-button dcf-btn-toggle-modal dcf-d-flex dcf-ai-center dcf-jc-between dcf-w-100% dcf-p-0 dcf-txt-2xs dcf-bg-transparent dcf-b-1 dcf-b-solid unl-b-scarlet unl-font-sans unl-scarlet" data-toggles-modal="dcf-search-results"></button>
   </div>
-  <div class="unl-search-modal dcf-fixed dcf-pin-top dcf-pin-left dcf-h-100% dcf-w-100% dcf-bg-overlay-light" id="dcf-search-results" aria-labelledby="unl-search-modal-heading" aria-hidden="true" role="dialog" tabindex="-1" hidden>
-    <div class="dcf-d-flex dcf-flex-col dcf-flex-nowrap dcf-h-100% dcf-w-100%" role="document">
-      <h2 class="dcf-sr-only" id="unl-search-modal-heading">Search Form</h2>
-      <div class="dcf-bleed dcf-bg-center dcf-bg-no-repeat dcf-bg-cover unl-bg-scarlet unl-search-bg">
+  <div id="dcf-search-results" class="dcf-modal dcf-bg-overlay-light dcf-fixed dcf-pin-top dcf-pin-left dcf-h-100% dcf-w-100% dcf-d-flex dcf-ai-center dcf-jc-center dcf-opacity-100 dcf-pointer-events-auto" hidden>
+    <div class="dcf-modal-wrapper dcf-h-max-100vh dcf-relative dcf-h-auto dcf-overflow-y-auto dcf-d-flex dcf-flex-col dcf-flex-nowrap dcf-h-100% dcf-w-100%">
+      <header class="dcf-modal-header dcf-bleed dcf-bg-center dcf-bg-no-repeat dcf-bg-cover unl-bg-scarlet unl-search-bg">
```

Comment:
Changing this from a `<header>` to a `<div>` should fix the Axe errors and still work with the DCF modal.

Reply:
done.